### PR TITLE
updated services, closes issue #174

### PR DESCRIPTION
**Pull request by theycallmemac**

### CODE REVIEW

File: source/help/services/index.md [hunk 1]
```diff
@@ -57,3 +59,28 @@ We currently offer 2 different database accounts:
...
+
+## Mastodon
+Mastodon is a federated social network which shares similarities to Twitter, but differs from Twitter in how it is administrated as a decentralized federation of servers running open source software. Basically Mastodon takes the best part from Twitter, and then it does it a little better.
+
+The distributed, open-source platform offers better tools for privacy and fighting harassment than Twitter offers, but coupled with this is a steep learning curve. Mastodon’s federated nature means there’s no single website to use, and learning how to sift through its timeline of toots can take some time.
+
+If you miss "the old Twitter”, i.e the days of purely chronological timelines and no ads, Mastodon is perfect for you! You can find Mastodon hosted on Redbrick at mastodon.redbrick.dcu.ie/.
+
+
+## hastebin
```
Comment (butlerx) [inline source/help/services/index.md]: capital

File: source/help/services/index.md [hunk 2]
```diff
@@ -57,3 +59,28 @@ We currently offer 2 different database accounts:
...
 Email Admins for a database account.
+
+
+## HackMD
+
+Redbrick hosts HackMD, a multiplatform collaborative markdown note editor which allows the creation of realtime markdown notes on all platforms. 
+
+It's real time that means you can write and edit notes with your peers at the same time. As you write, you can view he changes you make in HackMD's split screen view. Needless to say, it works very uniquely, all you have to do is to write an article over there and if you need any corrections or additions from your peers, then you can send them that article's web URL and they can edit and rewrite it as they think fit plus an original copy will always remain with you.
+
+Check out HackMD at md.redbrick.dcu.ie/.
```
Comment (butlerx) [inline source/help/services/index.md]: make this a link

File: source/help/services/index.md [hunk 3]
```diff
@@ -57,3 +59,28 @@ We currently offer 2 different database accounts:
...
+
+## Mastodon
+Mastodon is a federated social network which shares similarities to Twitter, but differs from Twitter in how it is administrated as a decentralized federation of servers running open source software. Basically Mastodon takes the best part from Twitter, and then it does it a little better.
+
+The distributed, open-source platform offers better tools for privacy and fighting harassment than Twitter offers, but coupled with this is a steep learning curve. Mastodon’s federated nature means there’s no single website to use, and learning how to sift through its timeline of toots can take some time.
+
+If you miss "the old Twitter”, i.e the days of purely chronological timelines and no ads, Mastodon is perfect for you! You can find Mastodon hosted on Redbrick at mastodon.redbrick.dcu.ie/.
```
Comment (butlerx) [inline source/help/services/index.md]: link plz

File: source/help/services/index.md [hunk 4]
```diff
@@ -57,3 +59,28 @@ We currently offer 2 different database accounts:
...
+
+
+## hastebin
+Ever hear of a pastebin? It's just like that, except you're in a hurry! hastebin is a service we host for storing and sharing text and code snippets with your peers.
+
+Paste your code, markdown or whatever you feel like pasting into the provided space. After that, just save it, get your link and then share it!
+
+Try out hastebin at paste.redbrick.dcu.ie.
```
Comment (butlerx) [inline source/help/services/index.md]: link please

File: source/help/services/index.md (hunk 2 above)
Comment (CSmartt) [reply source/help/services/index.md]: [I am a title] (https://stuff.things)

**Comment by theycallmemac** [comment]
I did done it

**Pull request merged**